Source/ChilliSource/Rendering/Base/RenderCommandCompiler.cpp

```diff
@@ -333,11 +333,11 @@ namespace ChilliSource
     }
     
     //------------------------------------------------------------------------------
-    RenderCommandBufferUPtr RenderCommandCompiler::CompileRenderCommands(const TaskContext& taskContext, const std::vector<TargetRenderPassGroup>& targetRenderPassGroups, RenderCommandListUPtr preRenderCommandList,
-                                                                          RenderCommandListUPtr postRenderCommandList, RenderFrameData renderFrameData) noexcept
+    RenderCommandBufferUPtr RenderCommandCompiler::CompileRenderCommands(const TaskContext& taskContext, IAllocator* frameAllocator, const std::vector<TargetRenderPassGroup>& targetRenderPassGroups, RenderCommandListUPtr preRenderCommandList,
```

Comment:
I thought that if you specified it as an rvalue ref type in the argument it enforced that it had to be moved. Whereas if you specify it as a value type there is no compiler help telling the caller to do an std::move